.github/workflows/build.yml

```diff
@@ -112,13 +112,14 @@ jobs:
           cabal update
           sed -i 's/^jobs:/-- jobs:/' "${HOME}/.cabal/config"
           cd "$GITHUB_WORKSPACE"
-          cabal v1-install --enable-tests --only-dep $CABAL_OPTS
 
       - name: script
         run: |
           cd "$GITHUB_WORKSPACE"
-          (cd ghci-wrapper && cabal v1-configure --enable-tests --ghc-options=-Werror $CABAL_OPTS && cabal v1-build && cabal v1-test)
-          cabal v1-configure --enable-tests --ghc-options=-Werror $CABAL_OPTS && cabal v1-build && cabal v1-test
+          (cd ghci-wrapper && cabal configure --enable-tests && cabal build && cabal test --test-show-details=direct)
+          cabal configure --enable-tests && cabal build && cabal exec -- cabal test --test-show-details=direct
```

Comment:
If `cabal test` needs to be under `cabal exec`, will `stack` be able to run the tests?

Reply:
I guess it will work with `stack exec` as well.  `doctest` is not really different from `ghci` in that regard.